### ISSUE TITLE
Craft 4 Support

### ISSUE THREAD

**Issue by lopugit** [JungleMinds/Craft_NetHooks#4]
Would it be possible to update so this shows as Ready in the Craft 4 Upgrade page.

Regards 

![Screen Shot 2022-05-16 at 10 27 58 am](https://user-images.githubusercontent.com/13629190/168501286-c3b99f61-e8cd-4bf2-a94d-271e322f5d80.png)

**Comment by theAdhocracy** [comment]
+1

**Comment by simon-onedotsix** [comment]
+1

**Comment by a-rbsn** [comment]
+1

**Comment by genio** [comment]
Is there anything we can do to help this along?

**Comment by genio** [comment]
@angrybrad Can you help us out here or is this no longer maintained by JungleMinds?

**Comment by angrybrad** [comment]
@genio not exactly sure what you're asking, but it appears there isn't a Craft 4 compatible version of this plugin yet.

**Comment by genio** [comment]
Right, and with @grumpyoldman-io no longer working with JungleMinds, the only other contributor I see is you. I'm basically just trying to knock on doors until we find the right person who can fix and cut releases of this plugin

**Comment by angrybrad** [comment]
If it's no longer being maintained, I can mark it as abandoned on https://plugins.craftcms.com, but we've got no control over the repo. The repo would need to be transferred to someone who wants to maintain it or someone could fork it and submit it as a new plugin to the Craft plugin store.

**Comment by a-rbsn** [comment]
Just looking at the forks and here:

https://github.com/thirstcreativedevelopers/nethooks

Is it as simple as updating composer to require craft 4 instead of 3?

**Comment by genio** [comment]
@grumpyoldman-io any help here?
